### ISSUE TITLE
rubocop gem error  in vagrant installation. 

### ISSUE THREAD

**Issue by arruah** [mysociety/alaveteli#5050]
When I run `vagrant provosion` I got the error: 
`   default: Gem::InstallError: rubocop requires Ruby version >= 2.2.0.  
    default: An error occurred while installing rubocop (0.60.0), and Bundler cannot default: continue.
    default: Make sure that `gem install rubocop -v '0.60.0'` succeeds before bundling.
`

**Comment by garethrees** [comment]
Ah, I think you might be using the default Debian Jessie OS? We're about to drop support for that (https://github.com/mysociety/alaveteli/issues/4782) and Ruby 2.1 (https://github.com/mysociety/alaveteli/issues/4542).

Try:

```
vagrant destroy
cp .vagrant.yml.example .vagrant.yml
sed -i 's/jessie/stretch/' .vagrant.yml 
vagrant provision
```

**Comment by gbp** [comment]
Fixed in dd8625cebd41ed6cf7af489ab0f2145031e34b0a
